File: assignment/areaofsquare.cpp
#include<iostream>
using namespace std;
class Area
{
    float num1,num2,area;
    public:void areaofsquare()
    {
        cout<<"Length of one side: \n";
        cin>>num1;
        cout<<"Length of the other side: \n";
        cin>>num2;
        area=num1*num2;
        cout<<"Area of the square is: \n"<<area;
    }
};
int main()
{
    Area obj;
    obj.areaofsquare();
}

File: Java assignment(1-2)/Java Assignment 1/Area_Of_Square.java
import java.util.Scanner;
class Area_Of_Square
{
float side1,side2,area;
void getdata()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter the length of side a: ");
side1=sc.nextFloat();
System.out.println("Enter the length of side b: ");
side2=sc.nextFloat();
}
void display()
{
area=(side1*side2);
System.out.println("The area of the square is: "+area);
}
public static void main(String...args)
{
Area_Of_Square obj=new Area_Of_Square();
obj.getdata();
obj.display();
}
}

